### DISCUSSION TITLE
any documentation about module.json file?

### DISCUSSION THREAD

**Discussion by sprklinginfo** [nWidart/laravel-modules#1446] [Q&A] [answered]
Hi, I couldn't find information about when/how the module.json is used (apologies if I have omitted it). just saw it is mentioned sparsely here and there in the repo's issues, e.g. 'files' key could be used to load php files in the module's directory. 
```php
{
    "name": "Blog",
    "alias": "blog",
    "description": "",
    "keywords": [],
    "priority": 0,
    "providers": [
        "Modules\\TenantAdmin\\Providers\\BlogServiceProvider"
    ],
    "aliases": {},
    "files": [],
    "requires": []
}
```
So what about other keys, like 'priority' or 'alias' or 'requires'? and for the 'providers' key here, does it have anything to do with `provides()` function in `BlogServiceProvider.php` file? thanks!

**Comment by dcblogdev** [comment, marked as answer]
`aliases` and `requires` are no longer used, I think they we're used in older versions.
`priority` is a way to set a loading order by giving different modules a different number.
I'm not sure if providers is used actually, its generated with the module.
